.gitlab-ci.yml

```diff
@@ -166,7 +166,7 @@ variables:
   DATADOG_AGENT_NIKOS_BUILDIMAGES: v18868373-918b265
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v18868373-918b265
-  DATADOG_AGENT_BUILDERS: v18823563-572898a
+  DATADOG_AGENT_BUILDERS: v19351184-76e2eef_test_only
```

Comment:
Will be updated as soon as https://github.com/DataDog/datadog-agent-builders/pull/239 is merged